packages/commons-server/test/suites/templating-helpers/helpers.spec.ts

```diff
@@ -199,6 +202,60 @@ describe('Template parser', () => {
     });
   });
 
+  describe('Helper: date', () => {
+    it('Should return an empty string if given the wrong amount of arguments', () => {
+      const parseResult = TemplateParser('{{date}}', {} as any, {} as any);
+      expect(parseResult).to.be.equal('');
+    });
+
+    it('Should return an empty string if given the wrong amount of arguments', () => {
+      const parseResult = TemplateParser(
+        "{{date '2022-01-01'}}",
+        {} as any,
+        {} as any
+      );
+
+      expect(parseResult).to.be.equal('');
+    });
+
+    it('Should return a date using a the default format', () => {
+      const parseResult = TemplateParser(
+        "{{date '2022-01-01' '2022-02-01' 'YYYY'}}",
+        {} as any,
+        {} as any
+      );
+
+      expect(parseResult).to.be.equal(
+        'Sat Jan 08 2022 08:39:00 GMT+0100 (Central European Standard Time)'
```

Comment:
We forgot to modify this too :) 

```suggestion
        '2022'
```